pkg/manifest.yml

```
@@ -3,6 +3,8 @@ node_version: v2.1.0
 name: python-mock-server
 upgrade: recommended
 # TODO: add hardware requirements
-# TODO: add plugin
+plugin:
+  - git: https://github.com/NethermindEth/mock-avs.git#main:plugin
```

We are supporting only one plugin per package, remove the `-` to avoid managing `plugin` as a YAML list
```suggestion
    git: https://github.com/NethermindEth/mock-avs.git#main:plugin
```